en/releases/centreon-commercial-extensions.md

```diff
@@ -5,59 +5,53 @@ title: Commercial Extensions
 
 ## Introduction
 
-You can find in this chapter all changelogs concerning **Centreon Commercial
-Extension**.
-
-> It is very important when you update your system to refer to this
-> section in order to learn about behavior changes or major changes that
-> have been made on this version. This will let you know the impact of
-> the installation of these versions on the features you use or the
-> specific developments that you have built on your platform (modules,
+You can find in this chapter all changelogs concerning **Centreon Commercial Extension**.
+
+> It is very important when you update your system to refer to this section in order to learn about behavior changes or
+> major changes that have been made on this version. This will let you know the impact of the installation of these
+> versions on the features you use or the specific developments that you have built on your platform (modules,
 > widgets, plugins).
 
 If you have feature requests or want to report a bug, please contact support.
 
 ## Centreon MAP
 
-### 21.04.0
+### 21.10.0
 
-- Compatibility with Centreon 21.04
+- Compatibility with other 21.10 components.
 
 ## Centreon BAM
 
-### 21.04.0
+### 21.10.0
 
-- Compatibility with Centreon 21.04
+- Compatibility with other 21.10 components.
 
 ## Centreon MBI
 
-### 21.04.0
+### 21.10.0
 
-- Compatibility with Centreon 21.04
+- Compatibility with other 21.10 components.
 
 ## Centreon Auto Discovery
 
-### 21.04.0
-
-#### Host discovery
+### 21.10.0
 
-- *Association* mappers are now named *property*.
-- Ability to link the discovered hosts to host groups, either already existing ones or new ones created on-the-fly.
-- Ability to link the discovered hosts to host categories, either already existing ones or new ones created on-the-fly.
-- Ability to link the discovered hosts to host existing host severities.
-- Property, macro, hostgroup and hostcategory mappers now support concatenation of either custom strings or discovered information.
-- UX alignment with Resource Status
-    - Clicking anywhere on a job's row now opens the side panel, not the job's result.
-    - Clicking on the contextual arrow leads to the job's result (*ie.* the discovered hosts).
+- Compatibility with other 21.10 components.
+- Host discovery now handles arrays of objects (Advanced Attributes) in the discovery results
+  - Can be used in conditions for every kind of mappers
+  - Can be used to name hostgoups, hostcategories
+  - Use cases: Cloud tags (Azure, AWS), network services discovered by Nmap, VMware IP addresses
+- A warning sign appears with a tooltip in front of the discovered hosts that could not be added by the discovery job because it already existed
```

Comment:
```suggestion
- A warning sign appears with a tooltip in front of the discovered hosts that could not be added by the discovery job because they already existed
```